packages/mco/utils/disaster-recovery.ts

```diff
@@ -55,3 +66,51 @@ export const matchApplicationsToSubstring = (
   });
   return valid;
 };
+
+export const isObjectRefMatching = (
+  objectRef: ObjectReference,
+  objectRefList: string[]
+) => objectRefList?.includes(objectRef?.name);
+
+export const filterDRPlacementRules = (
```

Comment:
```suggestion
export const getfilteredDRPlacementRuleNames = (
```
The name of the function should match the action it is performing.